### ISSUE TITLE
Possibly a simple addition: Add the ability to display Store apps usually present in "Apps" folder appear directly in the "All Programs" list

### ISSUE THREAD

**Issue by bonzibudd** [Open-Shell/Open-Shell-Menu#405]
For me, going into the All Programs list, and then going into another folder just to look for Store apps is a bit of a hassle. I attempted a solution involving replacing the Start Menu programs folder with the "Shell:appsFolder" location, however doing that does not display other items correctly. As for a folder which shows the Store apps alone, I could not find a specific location which included shortcuts to that which I could add to the menu. That is why I think that an addition to make the apps in the Open-Shell "Apps" folder appear alongside normal programs in the All Programs list would be a useful addition.

**Comment by ge0rdi** [comment]
Related to #14 .
It seems it would require a setting as not everyone will like such change.

**Comment by bonzibudd** [comment]
Yeah, I figured that a lot of people wouldn't like this, the default setting should stay as it is. Could it still be considered to add as a toggle, maybe in the "Main menu" section in settings?

**Comment by ge0rdi** [comment]
Sure, it is valid feature request.

**Comment by Delstrom2** [comment]
Sorry to resurrect an old thread, but I would like to bump and support this. At the very least, could you guide me to the specific file that contains the majority of the code that would need to be changed? I have some c++ experience, and I would be willing to look into seeing if I can tweak it myself, but I'm struggling to find exactly where in the code the different app shortcuts are organized. (I should be able to find my way around from there.)
